tests/Browser/BookingReviewTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Tests\Browser;
+
+use App\Models\BookingRequest;
+use App\Models\User;
+use App\Notifications\CommentNotification;
+use Database\Seeders\RolesAndPermissionsSeeder;
+use Database\Seeders\RoomSeeder;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Support\Facades\Mail;
+use Laravel\Dusk\Browser;
+use SlevomatCodingStandard\Helpers\Comment;
+use Tests\DuskTestCase;
+
+class BookingReviewTest extends DuskTestCase
+{
+    use DatabaseMigrations;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        (new RolesAndPermissionsSeeder())->run();
+        (new RoomSeeder())->run();
+        User::factory(1)->create()->first();
+        User::first()->assignRole('super-admin');
+    }
+
+    public function testUserCanViewBookingReviewPage()
+    {
+        $booking = BookingRequest::factory()
+            ->count(1)
+            ->hasReservations(random_int(1, 3))
+            ->create(["status" => BookingRequest::REVIEW])->first();
+
+        $this->browse(function (Browser $browser) use ($booking)
+        {
+            $browser->loginAs(User::first());
+            $browser->visit('/bookings/review')
+                ->clickLink('Open Details')
+                ->pause(5000)
```

Comment:
```suggestion
                ->pause(250)
```

As mentioned in other PRs, avoid long pauses unless absolutely necessary.

Reply:
I'd actually say use `$browser->waitFor...()` style helpers instead. less total wait time and still solid on ci.

Reply:
Good call, @danseremet if you can use a waitFor instead that would be best

Reply:
done